### DISCUSSION TITLE
官方openwrt-21.02本地编译在 luci -> application找不到

### DISCUSSION THREAD

**Discussion by Taosonyu** [UnblockNeteaseMusic/luci-app-unblockneteasemusic#102] [Q&A] [answered]
官方openwrt-21.02本地编译在 luci -> application找不到此项，已经git到本地了，进入make menuconfig里面搜unblockneteasemusic能看到相关的信息，可是在选项里面无法找到，麻烦大神解答一下，万分感谢，网上的各种方法我都试过了还是不行。

**Comment by 1715173329** [comment, marked as answer]
勾选 libusteam-openssl
